appveyor.yml

```diff
@@ -1,20 +1,49 @@
+# Notes:
+#   - Minimal appveyor.yml file is an empty file. All sections are optional.
+#   - Indent each level of configuration with 2 spaces. Do not use tabs!
+#   - All section names are case-sensitive.
+#   - Section names should be unique on each level.
+
 version: "{build}"
+
 os: Windows Server 2012 R2
+
+clone_folder: c:\go\src\github.com\ipfs\go-ipfs
+
 environment:
   GOPATH: c:\gopath
-  TEST_NO_FUSE: 1
-  TEST_VERBOSE: 1
-  TEST_SUITE: test_sharness_expensive
+  #TEST_NO_FUSE: 1
+  #TEST_VERBOSE: 1
+  #TEST_SUITE: test_sharness_expensive
+  #GOFLAGS: -tags nofuse 
```

Comment:
I don't think so, because those will be set as windows envs, and cygwin does not pull in all windows envs only common things like `PATH`